recipes/s5cmd/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "s5cmd" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/peak/s5cmd/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b756ac8c69175d04e972c3b4deb6b689986d4260fbe83e32caea52d1061995a0
+
+build:
+  script: go install -v .
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+
+test:
+  commands:
+    - s5cmd --help
+
+about:
+  home: https://github.com/peak/{{ name }}
```

Comment:
Thanks for taking a look, @BastianZim --- it's very appreciated!

> Just to check, have you analysed the licenses with https://github.com/google/go-licenses as well? That generally gives a good overview.

That's correct.

> This is not a blocker but it would be good to hardcode this instead of using a variable. Sometimes the bot gets tripped up on those.

Will push a fix!

Reply:
Yeah, it went pretty smoothly. It required some manual effort of cleaning out the exported data (some lingering files unrelated to the actual license/notice files), but it's definitely much better than manually searching for everything.